source/robot_model/include/robot_model/Model.hpp

```diff
@@ -27,17 +28,28 @@ class Model {
   std::vector<std::string> joint_names_;                                          ///< name of the joints in the model
```

Comment:
Not sure we need to create that seperately because we can just access it via robot_model_.names. And otherwise we should do it for all used public attributes of pinocchio::Model, for consistency